apps/judicial-system/backend/src/app/modules/notification/notification.config.ts

```diff
@@ -31,10 +31,9 @@ export const notificationModuleConfig = defineConfig({
         'COURT_OF_APPEALS_ASSISTANT_EMAILS',
         '',
       ),
-      policeInstitutionEmails: env.requiredJSON(
-        'POLICE_INSTITUTIONS_EMAILS',
-        {},
-      ) as {
+      policeInstitutionEmails: env.requiredJSON('POLICE_INSTITUTIONS_EMAILS', {
```

Comment:
Adding this so we can get the email in our local env without adding these props in local env